fcrepo-serialization/src/test/resources/indexable.rdf

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<rdf:RDF
+    xmlns:indexing="http://fedora.info/definitions/v4/indexing#"
+    xmlns:fedora="http://fedora.info/definitions/v4/repository#"
+    xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+
+<rdf:Description rdf:about="http://localhost:8080/fcrepo4/rest/foo">
+    <rdf:type rdf:resource="http://www.w3.org/ns/ldp#RDFSource"/>
+    <rdf:type rdf:resource="http://www.w3.org/ns/ldp#Container"/>
+    <rdf:type rdf:resource="http://fedora.info/definitions/v4/repository#Container"/>
+    <rdf:type rdf:resource="http://fedora.info/definitions/v4/repository#Resource"/>
+    <rdf:type rdf:resource="http://fedora.info/definitions/v4/indexing#Indexable"/>
+    <indexing:hasIndexingTransformation>default</indexing:hasIndexingTransformation>
```

Comment:
the `indexing:hasIndexingTransformation` is an extension vocabulary and not relevant here. I'd suggest removing it (along with the namespace decl above).